<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatCollectorDialog</class>
 <widget class="QDialog" name="StatCollectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Количество слов</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Число слов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="wordsAmountLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="sameCombListView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>231</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
